velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -93,7 +102,42 @@ class SubstraitVeloxPlanConverter {
     return lengths_;
   }
 
+  /// Looks up a function by ID and returns function name if found. Throws if
+  /// function with specified ID doesn't exist. Returns a compound
+  /// function specification consisting of the function name and the input
+  /// types. The format is as follows: <function
+  /// name>:<arg_type0>_<arg_type1>_..._<arg_typeN>
+  const std::string& findFunction(uint64_t id) const;
+
  private:
+  /// A function returning current function id and adding the plan node id by
```

Comment:
nit: adding -> incrementing, function id -> plan node ID

`Returns unique ID to use for plan node. Produces sequential numbers starting from zero.`

Reply:
Fixed in this PR: https://github.com/facebookincubator/velox/pull/1637.